CIP-0036/README.md

```diff
@@ -230,6 +230,10 @@ Fund 8:
  - added the `voting_purpose` field to limit the scope of the delegations.
  - rename the `staking_pub_key` field to `stake_credential` and `registration_signature` to `registration_witness` to allow for future credentials additions.
 
+Fund 10:
+- stipulated that `reward_address` must be a Shelley payment address, otherwise voting reward payments will not be recieved. 
+  - **Note:** up to Fund 9 `reward_address` was a Shelley rewards address; from Fund 10 onwards, it will be a normal payment address.  This will allow rewards to be paid directly from the designated Catalyst pot rather than requiring MIR transfers.
```

Comment:
Could we add a link to some Catalyst documentation that verifies this change?  If no link exists, could we add some background here about the Catalyst team's decision to do this?

Reply:
> This will allow rewards to be paid directly from the designated Catalyst pot

What is, the "designated Catalyst pot" ? I thought this pot was precisely called the "treasury" 😶 ?

Reply:
@KtorZ from my reading of the CIP currently, Fund 10 will allow (or be based on) paying from accounts that are _not_ the treasury.  But @Ryun1 there is still a need to document this change.  The Catalyst changes are usually documented on a Git Doc somewhere & I still believe we should have a link to this resolution in the CIP text.

Reply:
@KtorZ @rphair 
Working on fleshing out the motivation for this.

Reply:
@rphair I think there is a misunderstanding here.  Proj Catalyst did not make a decision to make this change.  Proj Catalyst was forced to by circumstance and directive.  IOG does not control the keys sufficient to perform MIR transfers, so its incapable of performing them.  The only documentation proj catalyst has on registration and delegation is CIP-15 and CIP-36.  The motivation therefore is "Project Catalyst would like to continue to have the option of paying voter rewards.  Now that MIR transfers are unavailable for that purpose, the only option is to change the reward address to a normal payment address.  This also has the added benefit that these formats can be utilized by other applications,  which is what the CIP-36 specifically envisioned when it added a voting purpose to the record.  In recognition that this change may cause a large number of previously registered voters to now have an 'invalid' registration,  the prudent course would seem to be not denying a voter the right to vote, simply because they would be unable to receive a voter reward through no fault of their own."

Reply:
thanks @stevenj, that was in fact what my understanding was... once we get confirmation @Ryun1 is done working on the Motivation as it appears in the CIP then I'll double check that all your points are clarified there.

Reply:
thanks for updates @Ryun1 but so far I'm only seeing the Motivation section of the CIP draft has had "address" changed to "payment address" & nothing else.  @stevenj's [comments](https://github.com/cardano-foundation/CIPs/pull/373#discussion_r1061365560) are now reflected in the original post of this PR but I don't see them in the CIP document yet... have I missed something?

Reply:
@rphair 

Apologies I was unclear, poor choice of words - I was trying to say that @stevenj's comments have been reflected in this PR's motivation.